### ISSUE TITLE
App violates the impersonation policy

### ISSUE THREAD

**Issue by iUmarov** [googlesamples/android-play-billing#99]
I followed README and released the app to alpha testing. But, App has been suspended by Google Play.
Message from Google Play:
"After review, Trivial Drive v2, package.name, has been suspended and removed from Google Play as a policy strike because it violates the impersonation policy. Your app is identical to an existing app on the Play Store."
I found many duplicates of this app on the Play market released to testing. What was wrong with my release? I followed all steps...

**Comment by gorgon** [comment]
I am sorry to hear this.
It was probably an error on Google Play side. Could you please submit a request and they will unblock your app?
Just input your information here: https://support.google.com/googleplay/android-developer/contact/appappeals

**Comment by iUmarov** [comment]
@gorgon I appealed to Google Play saying that it was educational sample app provided in this repository, but Google Play team rejected it by saying this:

"After further review, I'm not able to reinstate your app as it looks to be the same as an app that’s live on Google Play.

You can find the app here: https://play.google.com/store/apps/details?id=net.iboxmanstudio.trivialdrive&hl=en&e=-EnableAppDetailsPageRedesign

Please be aware that this is a violation of the Spam policy and Impersonation Policy.

If you can make changes to your app to bring it into compliance, you'll need to upload it using a new package name and a new app name. If you do publish a new version of your app, please make sure it complies with our policies. You may want to review the Developer Program Policies for additional guidance."

**Comment by pfmaggi** [comment]
Thanks for reporting this.
I'm closing this issue because this forum is for issues with the sample.

For this particular case it could be useful to use [Google Play Internal test track](https://support.google.com/googleplay/android-developer/answer/3131213?hl=en-GB).

If you think that this is a problem in the Billings library, please open a bug on the Billing public issue tracker: https://issuetracker.google.com/issues/new?component=311487&template=1014446

Please reopen if you still think this is an issue with the sample.
